src/features/pageObjects/components/PageObjList.tsx

```diff
@@ -105,31 +115,31 @@ export const PageObjList: React.FC<Props> = (props) => {
               {...(!isNil(activePanel) ? { activeKey: activePanel } : {})}
               onChange={(key) => setActivePanel([...key])}
             >
-              {pageObjects.map(({ id, name, url, locators, library }) => {
-                const elements = selectConfirmedLocators(state as RootState, id);
-                const isPageObjectNotEmpty = !!size(locators);
+              {pageObjects.map((pageObject) => {
+                const elements = selectConfirmedLocators(state as RootState, pageObject.id);
+                const isPageObjectNotEmpty = !!size(pageObject.locators);
```

Comment:
why you refused object destructuring here?

you can use destructuring only for needed vars:
```
 {pageObjects.map((pageObject) => {
                const { locators, id, url, name } = pageObjects;
                ....
```
and then use whole pageObject on line 138